action/sealedenvelope.go

```diff
@@ -184,17 +184,11 @@ func wrapStakingActionIntoExecution(ab AbstractAction, toAddr []byte, pb proto.M
 	}, nil
 }
 
-// Verify verifies the action using sender's public key
-func (sealed *SealedEnvelope) Verify() error {
+// VerifyPubKey verifies the action using sender's public key
+func (sealed *SealedEnvelope) VerifyPubKey() error {
```

Comment:
`Verify()` is good
it is verifying the signature using pubkey, not verify pubkey